cli/flox-rust-sdk/src/providers/publish.rs

```diff
@@ -346,10 +346,10 @@ pub fn check_build_metadata(
                 panic!("expected build to succeed");
             },
             build::Output::Stdout(line) => {
-                info!("stdout: {line}");
+                println!("{line}");
             },
             build::Output::Stderr(line) => {
-                info!("stderr: {line}");
+                eprintln!("{line}");
             },
```

Comment:
I expect this direct use of STDIN and STDOUT will break any progress spinners (when added) and might be a good reason to keep presentation in the CLI crate. We can revisit in https://github.com/flox/flox/issues/2862 though.